lib/main_common.pm

```diff
@@ -1611,7 +1611,7 @@ sub load_extra_tests {
             diag "unknown scenario for EXTRATEST value $test_name";
         }
     }
-    loadtest "console/consoletest_finish" if console_is_applicable;
+    loadtest "console/consoletest_finish" if console_is_applicable and get_var('EXTRATEST') !~ /wicked/;
```

Comment:
1. I guess this will give perl warnings about the use of undefined variables, should be `&& !check_var('EXTRATEST', 'wicked')`
2. the reason why that module would not be included is far from obvious. Please make it more obvious at least by providing a comment, better to encode that into the already existing method `console_is_applicable`

Reply:
may I see reasoning why wicked test got consoletest_finish ? For me situations looks really odd - no one from people responsible for wicked testing wanted to add consoletest_finish it just fell down on our heads and we need to explain why we removing it ?

Reply:
my comment about "undefined variables" still applies. Also I would very much prefer if the test schedule would look cleaner. Did you look into my recommendation to handle that in `console_is_applicable`? You know, there is the complaint about the test schedule not being very clear and I think exactly lines like these are not helping in that regard.